$NetBSD: patch-src_fmopl_c,v 1.2 2012/05/04 16:00:01 joerg Exp $

Redefine INLINE to cover all inline functions for compilers with C99 semantic.

--- src/fmopl.c~	2006-05-13 05:36:42.000000000 +0000
+++ src/fmopl.c
@@ -31,7 +31,7 @@
  * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
  */
 
-#define INLINE		__inline
+#define INLINE		static __inline
 #define HAS_YM3812	1
 
 #include <stdio.h>
